pkg/fleet/installer/setup/common/setup.go

```diff
@@ -137,6 +139,18 @@ func (s *Setup) Run() (err error) {
 		if err != nil {
 			return fmt.Errorf("failed to install package %s: %w", url, err)
 		}
+
+		if p.name == DatadogAgentPackage {
+			// Check if the installer exists in the package, copy ourselves at the right place if not
+			// This is temporary and will be removed after next release
```

Comment:
should be in the postinst no ? doesn't matter very much since we'll be able to drop it tho